debs/xenial/archivematica-storage-service/debian-storage-service/postrm

```diff
@@ -7,6 +7,7 @@ echo "postrm" $1
 if [ $1 = "remove" ]; then
     #clean up storage service directories
     rm -rf /usr/share/python/archivematica-storage-service
```

Comment:
Should I remove this line, @scollazo? I wasn't entirely sure.

Reply:
Cleaning files after removing the package  doesn't hurt, and avoids piling up space used after uninstalling the package